.github/workflows/Release.yml

```diff
@@ -15,6 +15,6 @@ jobs:
       - name: "Build, Tag, and Push Docker Image"
         run: |
           echo "${{secrets.DOCKER_HUB_TOKEN}}" | docker login -u gopalkrishnaps --password-stdin
-          docker build -t gopalkrishnaps/intellij-android:2021.1.1-30 -t gopalkrishnaps/intellij-android:latest .
+          docker build -t gopalkrishnaps/intellij-android:2020.1.1-30 -t gopalkrishnaps/intellij-android:latest .
```

Comment:
Any reason you're not using the latest 2020 version?

Reply:
Yes. AFAIK from [the release notes](https://developer.android.com/studio/releases#4.1-intellij-idea) Studio is still using 2020.1.